README.rst

```diff
@@ -117,6 +117,13 @@ If you're installing Horovod on a server with GPUs, read the `Horovod on GPU <do
 If you want to use Docker, read the `Horovod in Docker <docs/docker.rst>`_ page.
 
 
+Compiling
```

Comment:
Let's merge this in the the "Install" section above, and remove references to the read-the-docs page.  So just say "To compile Horovod from source, follow the industrutions in the `Contributor Guide <docs/contributors.rst>`_.

Then, we can add something similar in https://github.com/horovod/horovod/blob/master/docs/summary.rst, which will then link to the correct GitHub or read-the-docs page depending on where the users is viewing the docs from.

Reply:
I agree with this edit